Add Learning Lab page for Chainguard Libraries for Python (June 2025)

smythp opened the pull request:
## Description

Basic page for Chainguard Libraries for Python learning lab.

smythp committed:
Added Learning Lab page for Chainguard Libraries for Python (June 2025)

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

mosabua reviewed (commented):
I think this looks good but how do you want to proceed .. merge this now and then rejig the content once the video is uploaded and timestamped? Also .. if we publish before it should link to the registration page...

mosabua commented on content/software-security/learning-labs/ll202506.md:
```diff
@@ -0,0 +1,49 @@
+menu:
+  docs:
+    parent: "learning-labs"
+weight: 110
```
Does this place it on the top of the list and nav .. it should.. latest should be first..

smythp committed:
Added demo link and information

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Changes based on Manfred's suggestions

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp commented:
Added registration and made it reverse chronological,  changed topics heading. Let's publish now and I'll add the recording later since I think this is a nice page for the LL. I also added some basic demo stuff, link, and link to deck.

smythp committed:
One typo caught by my only friend (Claude)

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Apply suggestions from code review

Co-authored-by: Manfred Moser <manfred@simpligility.ca>
Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Added slides PDF and removed blank line

Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

smythp committed:
Update content/software-security/learning-labs/ll202506.md

Co-authored-by: Manfred Moser <manfred@simpligility.ca>
Signed-off-by: Patrick Smyth <patrick.smyth@chainguard.dev>

mosabua reviewed (approved):
Can we change the pdf filename to all lowercase and avoid the _ that is weirdly placed at the moment. Otherwise looks good now and we should ship. We will update tomorrow or later this week anyway
